Invalid bls type conversion

```
error="...0chain/code/go/0chain.net/chaincore/threshold/bls/dkg.go:58:18: cannot use bls.CurveFp254BNb (constant unknown with invalid type) as int value in argument to bls.Init
```

--- a/code/go/0chain.net/chaincore/threshold/bls/dkg.go
+++ b/code/go/0chain.net/chaincore/threshold/bls/dkg.go
@@ -55,7 +55,7 @@ var dkgSummaryMetadata *datastore.EntityMetadataImpl
 
 /* init -  To initialize a point on the curve */
 func init() {
-	err := bls.Init(bls.CurveFp254BNb)
+	err := bls.Init(int(bls.CurveFp254BNb))
 	if err != nil {
 		panic(fmt.Errorf("bls initialization error: %v", err))
 	}

--- a/code/go/0chain.net/core/encryption/bls0chain.go
+++ b/code/go/0chain.net/core/encryption/bls0chain.go
@@ -14,7 +14,7 @@ import (
 var GenG2 *bls.G2
 
 func init() {
-	err := bls.Init(bls.CurveFp254BNb)
+	err := bls.Init(int(bls.CurveFp254BNb))
 	if err != nil {
 		panic(err)
 	}

